contracts/handlers/ERCHandlerHelpers.sol

```diff
@@ -24,6 +24,14 @@ contract ERCHandlerHelpers is IERCHandler {
     // token contract address => is burnable
     mapping (address => bool) public _burnList;
 
+    // token contract address => decimals
+    mapping (address => Decimals) public _decimals;
```

Comment:
TODO: create a task to pack _contractWhitelist, _burnList and _decimals into a single mapping.